README.md

```diff
@@ -55,6 +55,12 @@ The flask app has now been altered so that it is now partially integrated to the
 
 -A Trello Account to access the Trello API (This is free)
 
--To set up the Trello enviromental variables (Key, Token, List IDs etc) either by the altering the DevOps-Course-Starter/.env file or creating a seperate config file with those variables inside
+-A .env file with the Trello Enviromental variables (Key, Token, List IDs etc) inside, as well as a Flask Secret Key as well as the server configuration
```

Comment:
A new developer should be able to find the expected names of these variables easily - either specified here or in the .env.template file which doesn't look to be up to date

Reply:
So do I need to alter or delete it?